homeassistant_cli/cli.py

```diff
@@ -188,6 +188,17 @@ def _default_token() -> Optional[str]:
         ' Example: ENTITY=entity_name, NAME=attributes.friendly_name'
     ),
 )
+@click.option(
+    '--columns-width',
+    default=None,
+    type=click.INT,
+    envvar='HASS_COL_WIDTH',
+    show_envvar=True,
+    help=(
+        'Columns custom width. '
+        'If specified truncates column values (default: auto)'
```

Comment:
default is None here, right ? ..and how do I actually specify that if I wanted to ?

Reply:
Was thinking something like this:
```
@click.option(
    '--columns-width',
    default=0,
    type=click.INT,
    envvar='HASS_COL_WIDTH',
    show_default=True,
    help=('Truncates column values (0: auto, -1: disable).'),
)
```
But I'm not pretty sure. Thoughts?

Reply:
I can't think of a better approach at the moment - implement it and update the PR and lets see.